Herebyfile.mjs

```diff
@@ -1153,7 +1157,24 @@ export const packNativePreviewExtensions = task({
 
         await $({ cwd: extensionDir })`npm run bundle`;
 
-        const version = getVersion();
+        let version = "0.0.0";
+        if (options.forRelease) {
+            // No real semver prerelease versioning.
+            // https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions
+            assert(options.setPrerelease, "forRelease is true but setPrerelease is not set");

```

Comment:
To be clear this is obviously temporary and any stable release would not do this